src/plugins/image/src/main/js/core/Utils.js

```diff
@@ -178,6 +179,19 @@ define(
       return url.match(/^(?:site:\/\/|\/)\w/) !== null;
     };
 
+    /**
+     * Helper method that removes the instance URL from a given source, in case it was added by the browser, i.e. when an internal image is repositioned
+     */
+    var removeOriginFromSrc = function (src) {
+      if (!window.location.origin) {
+        window.location.origin = window.location.protocol + "//" + window.location.hostname + (window.location.port ? ':' + window.location.port : '');
+      }
+
+      var origin = window.location.origin;
+
+      return src.replace(origin, "");
```

Comment:
Perhaps removing the origin from the beginning of the string would be more reliable in case it's an external URL that for some reason has this origin as a request parameter, for example if origin is `https://example.com` and the src is `https://another.com/get/image?name=something&for=https://example.com`.

Reply:
Good point 👍